compiler/damlc/lib/DA/Cli/Damlc.hs

```diff
@@ -139,7 +139,11 @@ cmdIde numProcessors =
         <$> telemetryOpt
         <*> debugOpt
         <*> enableScenarioServiceOpt
-        <*> optionsParser numProcessors (EnableScenarioService True) (pure Nothing)
+        <*> optionsParser
+              numProcessors
+              (EnableScenarioService True)
+              (pure Nothing)
+              (optionalDlintUsageParser True)
```

Comment:
in `cmdIde` this gives users full control over linting, which is enabled by default.